install.ts

```diff
@@ -0,0 +1,218 @@
+import {
+  Process,
+  args,
+  chmod,
+  copyFile,
+  env,
+  exit,
+  platform,
+  writeFile,
+  removeAll,
+  rename,
+  run,
+  makeTempDir
+} from "deno";
+
+import { mkdirp } from "https://deno.land/x/std/fs/mkdirp.ts";
```

Comment:
I read `deno.mkdir` is already recursive. Maybe this can be removed?

See: [https://github.com/denoland/deno_install/pull/21/files](https://github.com/denoland/deno_install/pull/21/files(

Reply:
@MarkTiedemann Wow, that's cool!
That makes `mkdirp` obsolete indeed.